src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIRealtimeExtensions.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading.Tasks;
 using Microsoft.Shared.Diagnostics;
 using OpenAI.RealtimeConversation;
+using static Microsoft.Extensions.AI.OpenAIModelMappers;
```

Comment:
I'd prefer if we avoided `using static` like this. I have a hard time reading the code to know where things come from, and end up scouring around looking for private methods.